.github/workflows/qhelp-pr-add-comment.yml

```diff
@@ -0,0 +1,57 @@
+# This workflow is the second part of the process described in
```

Comment:
Let's not rename this file. It can actually be used by other (future) jobs that need to post PR comments.